website/docs/r/ses_domain_identity.html.markdown

```diff
@@ -20,6 +20,8 @@ The following arguments are supported:
 
 In addition to all arguments above, the following attributes are exported:
 
+* `domain` - Name of the domain.
```

Comment:
I believe `domain` is intentionally omitted here since it is included in the arguments section above. 

> In addition to all arguments above, the following attributes are exported